common.go

```diff
@@ -20,12 +20,13 @@ const langError = "language unavailable"
 // DataUnits represents the character chosen to represent the temperature notation
 var DataUnits = map[string]string{"C": "metric", "F": "imperial", "K": "internal"}
 var (
-	baseURL      = "http://api.openweathermap.org/data/2.5/weather?%s"
-	iconURL      = "http://openweathermap.org/img/w/%s"
-	stationURL   = "http://api.openweathermap.org/data/2.5/station?id=%d"
-	forecastBase = "http://api.openweathermap.org/data/2.5/forecast/daily?%s=%s&mode=json&units=%s&lang=%s&cnt=%d"
-	historyURL   = "http://api.openweathermap.org/data/2.5/history/%s"
-	dataPostURL  = "http://openweathermap.org/data/post"
+	baseURL        = "http://api.openweathermap.org/data/2.5/weather?APPID=%s&%s"
```

Comment:
Not to nitpick but let's keep the appid lower case like the rest of the parameters.  It's lower case in examples on the OWM site as well.